Update available 3D formats page (#10555)
godotengine/godot-docs: tutorials/assets_pipeline/importing_3d_scenes/available_formats.rst

There are 3 ways to export glTF files from Blender:

- As a glTF binary file (``.glb``).
- As a glTF text-based file with separate binary data and textures (``.gltf``
  file + ``.bin`` file + textures).

> glTF binary files (``.glb``) is the smaller option. They include the mesh and

Reviewer: ```suggestion
glTF binary files (``.glb``) are the smaller option. They include the mesh and
```
Possibly? Not actually sure what is more grammatical here

Author: I'll go with "are" since files is plural

needs of game engines and shouldn't be used as-is. However, scenes exported with
the built-in Collada support may still work for simple scenes without animation.

> For complex scenes or scenes that contain animations we highly recommend using glTF
> instead.

Reviewer: ```suggestion
For complex scenes or scenes that contain animations it is highly recommended to
use glTF instead.
```
Possibly? "We" for the Godot/the manual authors reads weird to me.
